etc/poco/bundle/main.json

```diff
@@ -0,0 +1,52 @@
+{
+  "allow": [
+    {
+      "description": "Open version and healthcehck endpoints",
+      "paths": [
+        "/version",
+        "/healthcheck",
+        "/deepcheck"
+      ],
+      "methods": [
+        "GET"
+      ],
+      "principals": {
+        "open": {
+          "reason": "Allow all requests to Version and healthcecheck endpoints"
+        }
+      }
+    },
+		{
+			"description": "Open all endpoints used in the App logic",
+			"paths": [
+				"/",
+				"/github/**",
+				"/jira/**",
+				"/maintenance",
+				"/public/**",
+				"/session/**",
+				"/setup"
+			],
+			"methods": [
+				"GET","POST","PUT","DELETE"
+			],
+			"principals": {
+				"open": {
+					"reason": "Allow requests to the app endpoints because authentication is handled on the app side"
+				}
+			}
+		},
+    {
+      "description": "Allow service team to call API endpoints",
+      "paths": [
+        "/api/**"
+      ],
+      "methods": [
+        "GET","POST","PUT","DELETE"
+      ],
+      "principals": {
+				"staff":{"groups": ["micros-sv--github-for-jira-dl-admins"]}
+			}
+    }
+  ]
```

Comment:
TODO:
- [ ] Add per-env access to Pollinator Tests
- [ ] Add ASAP Authentication for Pollinator Test clients